packages/notionable/src/global/global-state.ts

```diff
@@ -56,6 +56,21 @@ const CACHE_HOSTNAME =
 
 export type NetworkLoadingState = 'Pending' | 'Loaded' | undefined;
 
+export type HistoricalTrading = Record<
```

Comment:
Is it possible to import this from the analytics server? Maybe you can move all the type definitions at the top of the `analytics-server` file into the `Definitions` file in the core-entities repo and then import from there.